pygfx/renderers/wgpu/shaders/pointsshader.py

```diff
@@ -57,6 +57,16 @@ def __init__(self, wobject):
         else:
             raise RuntimeError(f"Unknown color_mode: '{color_mode}'")
 
+        edge_mode = str(material.edge_mode)
+        if edge_mode == "inner":
+            self["edge_mode"] = "inner"
+        elif edge_mode == "outer":
+            self["edge_mode"] = "outer"
+        # elif edge_mode == "centered":
+        # elif edge_mode == "auto":
+        else:
+            self["edge_mode"] = "centered"
```

Comment:
Why not just make the default centered? It doesn't seem like auto actually does anything clever?

Reply:
i was following the color mode pattern, happy to change this, i got bored of the "auto" too by the end.